app/models/geoip_info.rb

```diff
@@ -0,0 +1,19 @@
+class GeoipInfo < OIDC::BaseModel
```

Comment:
Does it make sense to inherit from OIDC base model here? What about to move this "BaseModel" somewhere in generic namespace and rename to keep it clear what's inside?

Reply:
I can move `OIDC::BaseModel` to `app/application_model.rb` ?

Reply:
Yup, anything outside OIDC will work for me. I'm not sure about the name, since for DB backed models, there is `ApplicationRecord`. `ApplicationModel` seems ok.

Reply:
With this change, I think this is ready to go.